src/globus_sdk/_testing/data/flows/validate_run.py

```diff
@@ -0,0 +1,108 @@
+from responses import matchers
+
+from globus_sdk._testing.models import RegisteredResponse, ResponseSet
+
+from ._common import TWO_HOP_TRANSFER_FLOW_ID
+
+VALIDATE_RUN_SIMPLE_INPUT_BODY = {"param": "value"}
+
+VALIDATE_INVALID_SIMPLE_INPUT_BODY = {"foo": "bar"}
+
+VALIDATE_RUN_SIMPLE_SUCCESS_RESPONSE = {"message": "validation successful"}
+
+validate_simple_input_request = {
+    "body": VALIDATE_RUN_SIMPLE_INPUT_BODY,
+}
+
+validate_invalid_simple_input_request = {
+    "body": VALIDATE_INVALID_SIMPLE_INPUT_BODY,
+}
+
+
+RESPONSES = ResponseSet(
+    metadata={
+        "flow_id": TWO_HOP_TRANSFER_FLOW_ID,
+        "request_params": validate_simple_input_request,
+    },
+    default=RegisteredResponse(
+        service="flows",
+        path=f"/flows/{TWO_HOP_TRANSFER_FLOW_ID}/validate_run",
+        method="POST",
+        status=200,
+        json=VALIDATE_RUN_SIMPLE_SUCCESS_RESPONSE,
+        match=[
+            matchers.json_params_matcher(
+                params={"body": VALIDATE_RUN_SIMPLE_INPUT_BODY},
```

Comment:
Is there a reason not to write this as
```suggestion
                params=validate_simple_input_request,
```
?

As a related aside, I'm a little wary of our applying `match` usage to our SDK-provided testing responses (we originally didn't do it at all, then started doing it). It has proven problematic in some cases where you want "a" response but you don't care about or want to vary the contents of the request.
For example, if you want a test in a downstream application `test_frobulate_validates_run_input` with parametrized inputs to `frobulate()`, you may have dozens of _different_ calls to this API, and you would prefer that the default SDK response *always* matched so you can write your test more simply.
I'm considering shifting us to not have a match specified for `default` and allowing it only for the other responses, but that would need a broader discussion.